guides/messaging-high-availability-openshift.adoc

```diff
@@ -0,0 +1,254 @@
+ImageStream in your namespace:
+[source, bash, options="nowrap"]
+----
+oc create imagestream remote-helloworld-mdb
+----
+
+Now push your image into the ImageStream:
+
+[source, bash, options="nowrap"]
+----
+export REGISTRY="$(oc get routes -n openshift-image-registry default-route -o=jsonpath='{.spec.host}')"
+podman login --tls-verify=false -u admin  -p $(oc whoami -t) $REGISTRY
+podman tag localhost/remote-helloworld-mdb  $REGISTRY/$(oc config view --minify -o jsonpath='{..namespace}'
+)/remote-helloworld-mdb
+podman push --tls-verify=false $REGISTRY/$(oc config view --minify -o jsonpath='{..namespace}'
+)/remote-helloworld-mdb
+----
+
+=== Install WildFly operator
+
+First we need to install WildFly operator to OpenShift cluster. Because WildFly operator is not part of OpenShift OperatorHub
+by default, we need to add it first. Create file `community-catalog-source.yaml` with content:
+[source, yaml, options="nowrap"]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+name: operatorhubio-catalog
+namespace: openshift-marketplace
+spec:
+displayName: Community Operators
+grpcPodConfig:
+securityContextConfig: restricted
+image: quay.io/operatorhubio/catalog:latest
+publisher: OperatorHub.io
+sourceType: grpc
+updateStrategy:
+registryPoll:
+interval: 60m
+----
+
+Execute the following command to add the community catalog to the OperatorHub:
+
+[source, bash, options="nowrap"]
+----
+oc apply -f community-catalog-source.yaml
+----
+
+You can now install the WildFly Operator using the `wildfly-operator.yml` file:
+
+[source, yaml, options="nowrap"]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: subscription-wildfly-operator
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: wildfly-operator
+  source: operatorhubio-catalog
+  sourceNamespace: openshift-marketplace
+----
+
+and install it by:
+
+[source, bash, options="nowrap"]
+----
+oc apply -f wildfly-operator.yml
+----
+
+Now you should the installed operator in your namespace.
+
+=== Deploy application image using WildFly operator
+
+We can now configure the WildFly Operator to deploy a WildFly image with the messaging application. To do this, we need
+to specify the location of the AMQ 7 broker by setting the environment variables
+`JBOSS_MESSAGING_CONNECTOR_HOST` and `JBOSS_MESSAGING_CONNECTOR_PORT`:
+
+[source, yaml, options="nowrap"]
+----
+apiVersion: wildfly.org/v1alpha1
+kind: WildFlyServer
+metadata:
+  name: wildfly-remote-activemq
+spec:
+  applicationImage: "remote-helloworld-mdb:latest"
+  replicas: 1
+  env:
+  - name: JBOSS_MESSAGING_CONNECTOR_HOST
+    value: amq-broker-acceptor-0-svc
+  - name: JBOSS_MESSAGING_CONNECTOR_PORT
+    value: '61616'
+----
+
+=== Test the Application
+
+We're going to test the application by sending 5 messages to the `HELLOWORLDMDBQueue` to AMQ 7 broker by using servlet and will check that MDB consumed those messages from this queue:
```

Comment:
s/to AMQ/on the AMQ/g
s/using/using a/g
s/servlet and/servlet. Then we/g
s/that MDB/that the MDB/g
s/this queue/the queue/g